scripts/main/lychee.js

```diff
@@ -731,33 +863,45 @@ lychee.setMode = function (mode) {
 	header.bind_back();
 };
 
+/**
+ * @param {jQuery} obj
+ * @param {string} animation
+ *
+ * @returns {void}
+ */
 lychee.animate = function (obj, animation) {
-	let animations = [
+	const animations = [
 		["fadeIn", "fadeOut"],
 		["contentZoomIn", "contentZoomOut"],
 	];
 
-	if (!obj.jQuery) obj = $(obj);
-
 	for (let i = 0; i < animations.length; i++) {
 		for (let x = 0; x < animations[i].length; x++) {
-			if (animations[i][x] == animation) {
+			if (animations[i][x] === animation) {
 				obj.removeClass(animations[i][0] + " " + animations[i][1]).addClass(animation);
-				return true;
```

Comment:
The complete elimination of `return` will force this nested loop to run to completion even if an entry is found on the first iteration. It's a short loop though so I guess it won't make much difference, and it's not like it can match more than once...

Reply:
I assume I removed `return true` because of its return value.

Reply:
OK, I put a `return` back via a suggestion.